js/foam/u2/UpdateView.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+CLASS({
+  package: 'foam.u2',
+  name: 'UpdateView',
+  extends: 'foam.u2.View',
```

Comment:
UpdateDetailView has visual features.

It adds save and cancel buttons.  This decorator does not, and therefore shouldn't be a view, there is nothing visual to it, it's purely part of the model layer.